### PR TITLE
PullRequests1

### PR DESCRIPTION
**Pull request by eslee4478**
PullRequestsContents1

### CODE REVIEW

File: GitFile1.txt [hunk 1]
```diff
@@ -1 +1 @@
-Contents2
```
Comment (eslee3209) [inline GitFile1.txt:1]: check this line

Comment (eslee3209) [reply GitFile1.txt:1]: okay

**Comment by eslee4478** [comment]
What's wrong?

File: GitFile1.txt (hunk 1 above)
Comment (eslee4478) [inline GitFile1.txt:1]: What's wrong?

File: GitFile1.txt [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+Contents4
```
Comment (eslee3209) [inline GitFile1.txt:1]: also check this line

**Pull request merged**